vignettes/reuse.Rmd

```diff
@@ -35,6 +35,53 @@ You can document multiple functions in the same file by using either `@rdname` o
 It's a technique best used with care: documenting too many functions in one place leads to confusion.
 Use it when all functions have the same (or very similar) arguments.
 
+### `@rdname`
+
+Use `@rdname <destination>` to include multiple functions in the same page.
+The functions will be collected and share the same `@title`, `@description`, `@examples`, etc.
```

Comment:
> The functions will be collected and share the same `@title`, `@description`, `@examples`, etc.

Sentence reads a bit awkwardly to me, I think it's the passive voice? Maybe "roxygen will collect functions marked with the same `<destination>` and give them the same `@title`..."?